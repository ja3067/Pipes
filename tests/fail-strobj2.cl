x = "hello"
x[0] = "c"
