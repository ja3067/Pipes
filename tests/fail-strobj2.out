STypeError: invalid types (str, int) for list assignment
